tests/lib/nsxv_test.py

```diff
@@ -337,23 +385,45 @@ def testStrForinet6(self):
 
   def testTranslatePolicy(self):
     """Test for Nsxv.test_TranslatePolicy."""
-    # exp_info default is 2
     self.naming.GetNetAddr('NTP_SERVERS').AndReturn([nacaddr.IP('10.0.0.1'),
                                                      nacaddr.IP('10.0.0.2')])
     self.naming.GetNetAddr('INTERNAL').AndReturn([nacaddr.IP('10.0.0.0/8'),
                                                   nacaddr.IP('172.16.0.0/12'),
                                                   nacaddr.IP('192.168.0.0/16')])
     self.naming.GetServiceByProto.return_value = ['123']
 
-    exp_info = 2
     pol = policy.ParsePolicy(INET_FILTER, self.naming, False)
-    translate_pol = nsxv.Nsxv(pol, exp_info)
+    translate_pol = nsxv.Nsxv(pol, EXP_INFO)
+    nsxv_policies = translate_pol.nsxv_policies
+    for (header, filter_name, filter_list, section_id, terms) in nsxv_policies:
```

Comment:
Why did you change the underscore to a named variable? _ is used when a variable is unused. Please take a look at the following to understand how to mark a variable as unused.
https://google.github.io/styleguide/pyguide.html?showone=Lint#Lint

You should also think about why section_id is returned here. It seems weird